docs/reference/modules/snapshots.asciidoc

```diff
@@ -60,6 +60,9 @@ or
 GET /_snapshot/_all
 -----------------------------------
 
+The `ignore_unavailable` parameter can be used to controls whether to ignore if any specified snapshots are unavailable.
```

Comment:
This is in the wrong section of the documentation (The command above is GetRepositoriesRequest).